examples/demo/site/themes/bootstrap/assets/js/roc_auth.js

```diff
@@ -1,7 +1,8 @@
 var ROC_AUTH = ROC_AUTH || { };
 
-var loginURL = "/login";
-var logoutURL = "/logoff";
+var loginURL = "/basic_auth_login";
```

Comment:
at some point, we may need to generate part of this script, and eventually cache it on the server.
or generate a real file under a "files" folder.

food for thought  -> support files upload + files publishing